### ISSUE TITLE
Duplicate term?

### ISSUE THREAD

**Issue by mchibucos** [evidenceontology/evidenceontology#141]
Hi @nsuvarnaiari . I noticed you had created a term called "deletion mutation evidence" ECO:0005510. But I had previously created "deletion evidence" ECO:0001038. Is 5510 in use? Should it be obsoleted or merged into 1038? Thoughts?

**Comment by rctauber** [comment]
I think @nsuvarnaiari and I discussed this briefly before. ECO:0001038 is the one with a 'used in manual assertion' child, so that's most likely the one being used. We did a quick search to see if ECO:0005510 was in use and nothing turned up.

The other mutation terms have labels like 'x mutation evidence', so could we possibly merge ECO:0005510 into ECO:0001038 (has alt. ID) and change the label to 'deletion mutation evidence'?
